app/views/Dashboard/Overview/MapView/MapModal/index.tsx

```diff
@@ -293,8 +293,8 @@ function MapModal(props: ModalProps) {
             let uncertainRange;
 
             if (country.errorMargin) {
-                negativeRange = bound(country.indicatorValue - country.errorMargin, 0, 1);
-                positiveRange = bound(country.indicatorValue + country.errorMargin, 0, 1);
+                negativeRange = (bound(country.indicatorValue - country.errorMargin, 0, 1)) * 100;
+                positiveRange = (bound(country.indicatorValue + country.errorMargin, 0, 1)) * 100;
```

Comment:
We might need to use decimalToPercentage for this for consistency purposes.

Reply:
This has been implemented